test/doc-views/layout/TR.handlebars

```diff
@@ -0,0 +1,259 @@
+            </p>
+        {{/if}}
+        {{#if header.hr.show}}
+            <hr title="Separator for header">
+        {{/if}}
+    </div>
+    {{#if hr.show}}
+        <hr title="Separator for header">
+    {{/if}}
+
+    <section id="abstract" class="introductory">
+        <h2><span class="handlebars-data">{{abstract.abstractText}}</span></h2>
+        <p>This specification defines an API that provides the time origin, and current time in sub-millisecond resolution, such that it is not subject to system clock skew or adjustments.</p>
+        {{#if showWrongDateFormat}}
+            <h2 id="sotd">Status of This Document</h2>
+            <p>Hi! This date has the wrong format: 13/Jan/1998. And these two</p>
+        {{/if}}
+
+    </section>
+
+    <section id="{{sotd.id}}" data-link-for="Performance" class="introductory">
+        <h2><span class="handlebars-data">{{sotd.title}}{{! Status of This Document }}</span></h2>
+
+        <p><span class="handlebars-data">{{{sotd.emHTML}}}{{! This section describes the status of this document... }}</span></p>
+
+        {{#config.isRescinded}}
+        <p>
+            W3C has <span class="handlebars-data">{{sotd.rescindText1}}{{! chosen to rescind}}</span> the <a href="https://www.w3.org/TR/2017/REC-samplespec-20170101/">Sample Specification Recommendation</a> for the following reasons: [...list of reasons...]. For additional information <span class="handlebars-data">{{sotd.rescindText2}}{{! about replacement or alternative technologies}}</span>, please refer to the <a href="{{sotd.rescindLink}}">explanation of Obsoleting, Rescinding or Superseding W3C Specifications</a>.
+        </p>
+        {{/config.isRescinded}}
+
+        <p>
+            This document <span class="handlebars-data">{{sotd.extra1}}</span>was published by the <a href="{{sotd.WGLink}}"><span class="handlebars-data">{{sotd.group}}{{!Web Performance Working Group}}</span></a> as a <span class="handlebars-data">{{config.longStatus}} {{config.crType}}{{config.cryType}}</span> using the <a href='{{sotd.trackLink}}'><span class="handlebars-data">{{config.track}}</span> track</a>.
+        </p>
+
+        {{#config.isREC}}
+        <p>
+            W3C recommends the wide deployment of this specification as a standard for the Web.
+        </p>
+        {{/config.isREC}}
+
+        <p>
+            {{! checked by rule sotd.stability & sotd.draft-stability}}
+            <span class="handlebars-data">
+                {{! Publication as a xxx does not imply endorsement ... It is inappropriate to cite ...}}
+                {{> stability}}
+            </span>
+        </p>
+
+        {{#config.isCR}}
+            <p>This Candidate Recommendation is not expected to advance to Proposed Recommendation any earlier than
+                {{#if config.isEchidna}}
+                    <span class="handlebars-data">{{sixMonthLater}}.</span>
+                {{else}}
+                    04 October 2022.</p>
```

Comment:
This will fail starting from September 2022. It might be good to parameterize that date too.